src/mashupDocumentParser.ts

```diff
@@ -44,16 +43,16 @@ export default class MashupHandler {
         };
     }
 
-    private async editSingleQueryPackage(packageOPC: ArrayBuffer, queryName: string, query: string) {
+    private async editSingleQueryPackage(packageOPC: ArrayBuffer, queryMashupDoc: string) {
         const packageZip: JSZip = await JSZip.loadAsync(packageOPC);
         this.getSection1m(packageZip);
```

Comment:
this.getSection1m is returning the section but here it's not kept anywhere.. do you still need this call?
If it's only for checking that it's exists maybe we should rename the method?